_rules/important-letter-spacing-wide-enough-24afc2.md

```diff
@@ -16,74 +16,72 @@ input_aspects:
 acknowledgments:
   authors:
     - Jean-Yves Moyen
+  previous_authors:
     - Jey Nandakumar
   funding:
     - WAI-Tools
 ---
 
 ## Applicability
 
-This rule applies to any [HTML element][] that is [visible][] and for which the `style` attribute [declares][declared] the [letter-spacing][] CSS property.
+This rule applies to any [HTML element][] with one or more [visible][] [text node][] child, when all the following are true for the `letter-spacing` property of the element:
 
-## Expectation
+- the [specified][] value is [declared][] in a `style` attribute; and
+- the [computed][] value is [important][].
```

Comment:
Is the computed value `important`?
Or is it the declaration of the `letter-spacing` property that is `important`?
The [`important` declaration](https://www.w3.org/TR/css-cascade-4/#important) influences how the computed value is obtained, but I'm not sure we can say the computed value is `important`....

Reply:
Good point. We do need to somehow look at the computed value due to Inapplicable Example 8, where the `inherit !important` declaration is important but we don't care because the computed value is the inherited one, coming from a non-important declaration and thus can be overwritten.
I do agree that only declarations seems to be important 🙈 

Given that this wording is already present in [the current rule](https://www.w3.org/WAI/standards-guidelines/act/rules/24afc2/) (first bullet in Expectation + Passed Example 7), I suggest we treat that in another Issue/PR: https://github.com/act-rules/act-rules.github.io/issues/2002

Reply:
I've added a comment on https://github.com/act-rules/act-rules.github.io/issues/2002